pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -855,86 +854,3 @@ func bridgeStrToNum(bnStr string) (int, error) {
 	}
 	return bnNum, nil
 }
-
-func flowTimeOutSet(item string, origValue int) {
-
-	tOutValue := origValue + int(timeoutSec) // add 150 seconds
```

Comment:
This key part of the logic has gone missing; a comment saying "update" isn't the same as specifying the relationship between the timer values i.e., that timeoutSec needs to be added to a particular subset of the sysctl values.

Reply:
Can you suggest a better comment? Note that I would agree with you if you were reading the current values and adding 150 to them. The way it is implement -- is no better or worse compared to what this PR is doing.

Reply:
@naiming-zededa is the one that came up with the scheme where we need to add that constant. Can you check with him if he has suggestions on how to handle this?

Reply:
Let me check with @naiming-zededa today (now that I have a bit more spare time). Additionally, my interest in pulling all this into a separate container that was specifically designed to handle sysctl setting is that:
   1. it has higher chance of working out of the box (compared to us trying to tweak pillar container settings, etc.)
   2. if it doesn't -- we have the linuxkit community to help us figure out why not

At the same time I agree that some of the more involved logic (like the one for StrongSWAN) is not easy to be converted into this style -- so we'll punt on those for now. It is just that this one is a borderline that I feel like we can pull into a specially designed container.